go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/docker/compose/v2
 
-go 1.21.0
+go 1.22.0
 
-toolchain go1.22.2
+toolchain go1.22.7
```

Comment:
mostly curious; have you tried if it allows removing this one? I know it's sometimes added automatically, but if you remove it, it doesn't add it back